vscode-extension/src/extension.ts

```diff
@@ -9,6 +9,7 @@ interface IFoundError {
   language: string; // Language error found in
   rawText: string; // Entire blob of error message
   title: string; // Best title to show
+  blobId?: number; // Id of the individual blob containing error
```

Comment:
I don't think it's a good idea to assume a 1:1 relationship between an error and a blob of incoming text from tail. We're pretty sure there will be future cases where we're getting only one line at a time. Also related, the webview shouldn't "know" about blobs at all (as in `main.js:8`)—that should be abstracted into the concept of an "error". (Or "problem", or whatever we call these things. 😉)

Reply:
I was thinking we would group various errors under one blob, if there were several.
This way, we could still display questions and answers and issues and other stuff, even if our regexes don't detect an error per se.
I.e. I think there are situations where we can't detect an identifiable error, but still could be helpful

Reply:
Hmm, I'm thinking of the opposite case where one error spans multiple blobs. E.g. the error+stacktrace amount to 20 lines, which arrive as 20 separate blobs. (Right now that doesn't happen, but it's almost guaranteed to happen with some combination of language/compiler/ssh/remote/tailing/etc..)

In general, I think the concept of "blob" should be restricted to the case where we're trying to identify an error in a stream. Once we've identified an error, then create an error object (`IFoundError` at the moment) and from then on refer to it by its own id.